tools/run_tests/run_tests.py

```diff
@@ -1583,14 +1583,15 @@ def git(cmd, cwd=cwd):
         child_argv[1:])
 
     env = os.environ.copy()
-    env['RUN_TESTS_COMMAND'] = run_tests_cmd
     env['DOCKERFILE_DIR'] = dockerfile_dir
-    env['DOCKER_RUN_SCRIPT'] = 'tools/run_tests/dockerize/docker_run_tests.sh'
+    env['DOCKER_RUN_SCRIPT'] = 'tools/run_tests/dockerize/docker_run.sh'
+    env['DOCKER_RUN_SCRIPT_COMMAND'] = run_tests_cmd
+    # TODO(jtattermusch): is the XML_REPORT env variable any useful?
     if args.xml_report:
         env['XML_REPORT'] = args.xml_report
```

Comment:
Is this the required argument that make "run_tests.py" generate xml reports for Kokoro to consume?

Reply:
AFAICT this is unnecessary since the `run_tests_cmd` already contains all the arguments required by the child running inside docker container (including the `-x` argument used for reports). Based on my knowledge, the XML_REPORT variable is unused.
I will will remove in a followup PR to not risk breaking something.